package.json

```diff
@@ -29,7 +29,8 @@
   "dependencies": {
     "@actions/core": "1.2.6",
     "@actions/github": "4.0.0",
-    "glob": "7.1.6"
+    "glob": "7.1.6",
+    "wildcard-match": "5.1.0"
```

Comment:
memfs seems to be significantly a bigger lib than wildcard-match.

https://bundlephobia.com/result?p=wildcard-match@5.1.0
<img width="773" alt="Screenshot 2021-05-11 at 16 55 56" src="https://user-images.githubusercontent.com/1938043/117837471-c4f35380-b279-11eb-9a69-ab7317752493.png">
https://bundlephobia.com/result?p=memfs@3.2.2
<img width="834" alt="Screenshot 2021-05-11 at 16 56 19" src="https://user-images.githubusercontent.com/1938043/117837531-d0467f00-b279-11eb-8077-fec2d07c7898.png">

Reply:
I see, and `wildcard-match` is used only for virtual paths? Can it replace usage of glob?

Reply:
wildcard-match does not have file system functionality, it would require to  implement a walk function and the function `hasMagic` from glob basically recreating glob from scratch.

I tried to find other libraries that might habe file system capability but none of these has it as it can be seen in the table.

<img width="869" alt="Screenshot 2021-05-11 at 17 30 27" src="https://user-images.githubusercontent.com/1938043/117940250-2b738280-b309-11eb-8b81-b0db9afdbf71.png">

There are other alternatives to glob like fast-glob or tiny-glob but they can only do file system  operations.

In my opinion the cost of adding a library to the action is not so high since the code is run in GitHub and is never downloaded to the client.

Reply:
okay, let's use both